docs/conf.py

```diff
@@ -47,6 +47,7 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
+autoclass_content = 'both'
```

Comment:
Can we add a short in-line comment for this? That way we can go back and see why this choice was made without having to Google. Thanks!